tests/sample_outputs/run_attack_from_file.txt

```diff
@@ -1,4 +1,4 @@
-/.*/Attack(
```

Comment:
You need to add this /.*/ back in for the test to pass. 

It's the regex for "any character" and helps us match the text in Github Actions, since there's a bunch of extra stuff printed out when this attack is run on the VM for the first time.

Reply:
Oh interesting. The tests were successful on my VM so I went ahead with it .